docs/high-performance-order-storage/guide-large-store.md

```diff
@@ -0,0 +1,134 @@
+
+At the very least, we suggest you focus on the following tests:
+
+1. Test checkout with every payment method.
+2. Test refunds for those checkouts.
+3. If you are using WooCommerce Subscriptions, check that subscriptions can be purchased and renewals come through.
+4. Any critical flows you’ve defined for your site.
+
+You can repeat the tests with both synchronization enabled and disabled.
+
+Remember that you can [toggle synchronization](https://woocommerce.com/document/high-performance-order-storage/#section-5) by going to **WooCommerce > Settings > Advanced > Features**, and toggling the “**Enable compatibility mode**” checkbox.
+
+![Toggle synchronization](https://woo.com/wp-content/uploads/2023/10/image-18.png)
+
+**Review: Phase 1 Checklist**
+
+- [ ] Make sure code on your test site is up to date by grabbing the latest versions of all WooCommerce-related plugins you’re running.
+- [ ] Test checkout functionality for every payment method.
+- [ ] Verify refund process for test checkouts.
+- [ ] If WooCommerce Subscriptions is installed, test subscription purchases and renewal.
+- [ ] Test critical site flows with synchronization both enabled and disabled.
+
+## Phase 2: Migrations and testing on a staging site
+
+When you’re happy with how things are working in your local environment, it is time to copy over your production database to the staging site and test out HPOS migrations. For large sites, we recommend setting synchronization to on but, instead of relying on the scheduled jobs to perform the migration, you can speed things up by using the CLI: specifically, the command **wp wc cot sync**.
+
+To have a sense of how much the final migration (on production) is going to take, make sure to time how much it takes for this command to run. For example, for our test store of 9 million orders, this took about a week to complete.
+
+For extra peace of mind, you can run  **wp wc cot verify_cot_data --verbose** to check that the data has migrated successfully.
+Once migration is complete on your staging site, and still with synchronization enabled, re-run the tests defined in Phase 1. **Disable the sync, and go through all testing again.**
+
+### Test out 3rd party and external systems
+
+You may have non-PHP systems directly connected to DB that would escape a normal code audit. Verify that if you have systems such as data warehouses, shipping tracking systems, accounting systems, etc., they are not reading the posts tables directly. If they are, then they would need to be updated before you can proceed with the next steps.
+
+**Review: Phase 2 Checklist**
+1. [ ] Make sure code on your staging site is up to date by grabbing the latest versions of all WooCommerce-related plugins you’re running.
```

Comment:
Make sure `the` code on your staging site is up to date by `installing` the latest versions of all WooCommerce-related plugins you’re running.